src/bin/hexyl.rs

```diff
@@ -104,24 +104,46 @@ fn run() -> Result<(), AnyhowError> {
                 .takes_value(true)
                 .value_name("WHEN")
                 .possible_values(&["always", "auto", "never"])
+                .default_value_if("plain", None, "never")
                 .default_value("always")
                 .help(
                     "When to use colors. The auto-mode only displays colors if the output \
                      goes to an interactive terminal",
                 ),
         )
+        .arg(
+            Arg::with_name("plain")
+                .short("p")
+                .long("plain")
+                .help("Display output with --no-chars, --no-position, and --border=none."),
+        )
         .arg(
             Arg::with_name("border")
                 .long("border")
                 .takes_value(true)
                 .value_name("STYLE")
                 .possible_values(&["unicode", "ascii", "none"])
+                .default_value_if("plain", None, "none")
                 .default_value("unicode")
                 .help(
                     "Whether to draw a border with Unicode characters, ASCII characters, \
                     or none at all",
                 ),
         )
+        .arg(
+            Arg::with_name("no_chars")
+                .short("C")
+                .long("no-chars")
+                .conflicts_with("plain")
```

Comment:
Does this have to be conflicting with `--plain`? It's not really necessary, right?

Reply:
I suppose not, though it would be good to avoid using them together since that's redundant.

Reply:
True, it's redundant. But imagine someone typically wants to use `hexyl` without the char. table or the position and sets an alias:
```bash
alias hexyl="hexyl --no-file-position"
```

Then they would still be able to use `hexyl --plain` if they want to.